src/ResourceManager/Automation/Commands.Automation/Cmdlet/StartAzureAutomationDscCompilationJob.cs

```diff
@@ -49,6 +54,16 @@ public class StartAzureAutomationDscCompilationJob : AzureAutomationBaseCmdlet
         public IDictionary ConfigurationData { get; set; }
 
         /// <summary>
+        /// Gets or sets switch parameter to confirm building a new build version of the NodeConfiguration.
+        /// </summary>
+        [Parameter(Mandatory = false, HelpMessage = "Creates a new Node Configuration build version.")]
+        public SwitchParameter IncrementNodeConfigurationBuild
+        {
+            get { return this._incrementNodeConfigurationBuild; }
```

Comment:
@vrdmr you shouldn't need a separate value to track the value of a `SwitchParameter` type parameter. You should just be able to do `IncrementNodeConfigurationBuild.IsPresent` to determine whether or not the user provided this parameter.